src/Potato/Data/Text/Zipper2.hs

```diff
@@ -76,6 +76,12 @@ mapZipper f (TextZipper lb b s a la) = TextZipper
   , _textZipper_linesAfter = fmap (T.map f) la
   }
 
+appendEnd :: [Text] -> Text -> [Text]
```

Comment:
very easy to read this function and think it's [snoc](https://hackage.haskell.org/package/extra-1.7.12/docs/Data-List-Extra.html#v:snoc)

maybe give it a more descriptive name